2-Browsers/Week1/homework/ex3-hijackLogo.js

```diff
@@ -7,6 +7,9 @@
 ------------------------------------------------------------------------------*/
 function hijackGoogleLogo() {
   // TODO your code goes in here
+  const googleLogo = document.querySelector('.lnXdpd');
```

Comment:
This class `.lnXdpd` looks quite random. What would happen if google uploads a new version where this class is changed? 

Could you write a different selector that is less likely to change?

Reply:
I think this:
`
const googleLogo = () => {
    let tempImages = document.querySelectorAll('img');
    tempImages.forEach((element) => {
      if (element.alt === 'Google') {
        return element;
      }
    });
  };
`
But I'm eager to know your opinion.